pkg/config/manifest.go

```diff
@@ -546,6 +546,24 @@ func (m *Manifest) SliceElem(index int, val reflect.Value) error {
 		return nil
 	}
 
+	// There are two cases possible in the YAML. The first is when the element in the slice is a string. That might look like:
+	// install:
+	//   - description: "Install Hello World"
+	//     exec:
+	// 	     command: bash
+	// 	     arguments:
+	// 	       - -c
+	// 	       - "source:  bundle.parameters.command"
```

Comment:
I don't want to make this harder, because I agree that we need to rethink how we do sourcing anyway. But when I look at the two here,

- "source: bundle.parameters.command"
- source: bundle.parameters.command

My reaction is that the first one shouldn't even be parsed by porter and should be ignored really. It's only the second that matches how source works everywhere else.

Does that simplify this PR? If so, can we nix the first case "source as a string"? If it makes it harder, then leave it as it. 😀